content/en/about/logo.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Logo"
+draft: false
+---
+
+<img src="/images/logo-big.png" class="img-fluid logo-home pr-5" style="float: left;">
+
+The [InnerSource Commons logo](/images/logo-big.png) is a swirling, white-and-black vortex with the front part in the rough shape of the letter "I" (for InnerSource).
+It is licensed under `<INSERT LICENSE HERE>`, which means that you can modify and reuse it for your purposes.
```

Comment:
Left this pull request in draft because I'm not sure of what to put for `<INSERT LICENSE HERE>`.

Reply:
Something like the [Creative Commons Attribution Sharealike 4.0 International license](https://creativecommons.org/licenses/by-sa/4.0/)? This allows modification and use/redistribution by the end user under the same license, even allowing commercial usage as long as the user attributes the InnerSource Commons as the originator of the license.